tests/unittests/graphOptzTest.cpp

```diff
@@ -1554,6 +1554,29 @@ TEST_F(GraphOptz, foldQuantizeIntoVarMultipleUsages) {
   }
 }
 
```

Comment:
Can you also add a doxygen comment for the test?

Reply:
Done.